Core/Utilities/ElasticSearchDB.py

```diff
@@ -0,0 +1,279 @@
+  def setClusterName( self, name ):
+    """
+      It is used to set the cluster name
+      
+      :param self: self reference
+      :param str requestName: request name
+    
+    """
+    self.__clusterName = name
+  
+  ########################################################################
+  def getClusterName( self ):
+    """
+    It returns the cluster name
+    
+    :param self: self reference
+    """
+    return self.__clusterName
+  
+  ########################################################################  
+  def query( self, index, query ):
+    """It exexutes a query and it returns the result
+    query is a dictionary. More info: search for elasticsearch dsl
+    
+    :param self: self reference
+    :param dict query: It is the query in ElasticSerach DSL language
+     
+    """
+    return self.__client.search( index = index, body = query )
+  
+  def _Search( self, indexname ):
+    """
+    it returns the object which can be used for reatriving ceratin value from the DB
+    """
+    return  Search( using = self.__client, index = indexname )
+  
+  ########################################################################
+  def _Q( self, name_or_query = 'match', **params ):
+    """
+    It is a wrapper to ElasticDSL Query module used to create a query object. 
+    :param name_or_query str is the type of the query
+    """
+    return Q( name_or_query, **params )
+  
+  def _A( self, name_or_agg, aggsfilter = None, **params ):
+    """
+    It us a wrapper to ElasticDSL aggregation module, used to create an aggregation
+    """
+    return A( name_or_agg, aggsfilter, **params )
+  ########################################################################
+  def __tryToConnect( self ):
+    """Before we use the database we try to connect and retrive the cluster name
+    
+    :param self: self reference
+         
+    """
+    try:
+      if self.__client.ping():
+        result = self.__client.info()
+        self.setClusterName ( result.get( "cluster_name", " " ) )
+        self.log.info( "Database info", result )
+        self._connected = True
+      else:
+        self._connected = False
+        self.log.error( "Cannot connect to the database!" )
+    except ConnectionError as e:
+      self.log.error( e )
+      self._connected = False 
+
+  ########################################################################
+  def getIndexes( self ):
+    """
+    It returns the available indexes...
+    """
+    return [ index for index in self.__client.indices.get_aliases() ]
+  
+  ########################################################################
+  def getDocTypes( self, indexName ):
+    """
+    :param str indexName is the number of the index...
+    :return S_OK or S_ERROR 
+    """
+    result = []
+    try:
+      result = self.__client.indices.get_mapping( indexName )
+    except Exception as e:
+      gLogger.error( e )
+    doctype = ''
+    for i in result:
+      if not result[i].get( 'mappings' ) :
```

Comment:
Are you just trying to test the existance, or also that it is not a None/zero value ? If just the existence, better to use the following syntax
```if 'mappings' not in result[i]```